models/platypus_finance/avalanche_c/platypus_finance_avalanche_c_trades.yml

```
@@ -0,0 +1,83 @@
+version: 2
+
+models:
+  - name: platypus_finance_avalanche_c_trades
+    meta:
+      blockchain: avalanche_c
+      sector: dex
+      project: platypus_finance
+      contributors: umer_h_adil
+    config:
+      tags: ['avalanche_c', 'trades', 'dex', 'umer_h_adil']
+    description: >
+        Platypus Finance trades (which is only on Avalanche C-Chain). This table loads into dex.trades downstream.
+    columns:
+      - &blockchain
+        name: blockchain
+        description: "Blockchain which the DEX is deployed"
```

You can add an uniqueness test here.
```suggestion
    description: >
        Platypus Finance trades (which is only on Avalanche C-Chain). This table loads into dex.trades downstream.
    tests:
      - dbt_utils.unique_combination_of_columns:
          combination_of_columns:
            - block_date
            - blockchain
            - project
            - version
            - tx_hash
            - evt_index
            - trace_address
    columns:
      - &blockchain
        name: blockchain
        description: "Blockchain which the DEX is deployed"
```

---

Done